p2p/transport/websocket/listener.go

```diff
@@ -29,6 +29,9 @@ type listener struct {
 	// so we can't rely on checking if server.TLSConfig is set.
 	isWss bool
 
+	allowForwardedHeader bool
+	trustedProxies       []*net.IPNet
```

Comment:
nit: prefer `Prefix`, `Addr`, and `AddrPort` from the `netip` package. Generally fewer allocations and memory friendly.